tests/unit/transformers/test_base.py

```diff
@@ -1,10 +1,374 @@
+from unittest.mock import patch
+
 import pandas as pd
+import pytest
 
 from rdt.transformers.base import BaseTransformer
 
 
 class TestBaseTransformer:
 
+    def test_get_subclasses(self):
+        """Test the ``get_subclasses`` method.
+
+        Setup:
+            -
+
+        Input:
+            -
+
+        Expected behavior:
+            -
+        """
+        pass  # TODO: not sure how to test this one.
+
+    def test_get_input_type(self):
+        """Test the ``get_input_type`` method.
+
```

Comment:
Let's add a specific description of what is expected here. For example:

 "The base `get_input_type` method should return the value defined in the `INPUT_TYPE` of the child classes".